lib/presentation/mixins/contacts_view_controller_mixin.dart

```diff
@@ -402,49 +402,51 @@ mixin class ContactsViewControllerMixin {
                 .toList();
             if (presentationRecentContactNotifier.isDisposed) return;
 
-            if (Matrix.of(context).twakeSupported) {
-              if (keyword == null) {
-                presentationRecentContactNotifier.value =
-                    recent.take(_defaultLimitRecentContacts).toList();
-                return;
-              }
-              presentationRecentContactNotifier.value = [];
+            if (keyword == null || keyword.isEmpty) {
```

Comment:
Test cases:
- not existed recent item in tom, expect `presentationRecentContactNotifier` have the same list response from `_searchRecentChatInteractor`
- recent list have one item duplicated in tom contact, expect `presentationRecentContactNotifier` have list but not include duplicated item